### DISCUSSION TITLE
How to delete my Web3Auth account?

### DISCUSSION THREAD

**Discussion by chaitanyapotti** [Web3Auth/Web3Auth#326] [Feature Requests]
How to delete my Web3Auth account?

**Comment by himanshuchawla009** [comment]
No, You can't delete a web3auth account. You can't do this because of the following reasons:-

-  Web3Auth works on Cryptography primitives (Shamir secret sharing) and no private key data is being stored on the backend. 
-  Also all the reconstruction happens on the front end.

Looking at the above two properties we can conclude that, given the same inputs (shares/access factors), even in an offline manner the output will be the same coz of the cryptographic algorithms being used. So you will always be able to access your account and there is no way to delete your Web3Auth account.

You can read more about how Web3Auth works here:- 

https://web3auth.io/docs/overview/key-management/technical-architecture/
